### PR TITLE
add google tag manager

### PR DESCRIPTION
**Pull request by Andrey1994**

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ module.exports={
         },
         "gtag": {
```
Comment (retiutut) [inline website/docusaurus.config.js:36]: We should also remove gtag as part of this change.

**Review by retiutut — CHANGES_REQUESTED** [review]

**Comment by retiutut** [comment]
But we likely need the google tag manager plugin.

**Comment by Andrey1994** [comment]
why

**Comment by retiutut** [comment]
Gtag is the old analytics Gtm is the new one.

**Comment by Andrey1994** [comment]
And you've said to remove old one, so I did exactly that

File: website/package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.4.3",
+    "@docusaurus/plugin-google-tag-manager": "^2.4.3",
```
[the remark this answers is not in the document]
Comment (Andrey1994) [reply website/package.json]: If you use the preset @docusaurus/preset-classic, you don't need to install this plugin as a dependency.

You can configure this plugin through the preset options. 

from docusaurus docs https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-tag-manager and its exactly the case here

**Review by retiutut — APPROVED** [review]

**Pull request merged**